### PR TITLE
Replace felinid species with anime trait

### PR DESCRIPTION
**Pull request by lmenvs**
## About The Pull Request
This has been a point of debate a lot and if it is going to happen it has to happen before launch.  So I'm doing this to get it out there.

## Why It's Good For The Game
Felinid has always been snowflake code that started as a joke and... well, has never stopped being one.  This removes them as a "species" and makes them into a zero point cost neutral trait.  The way the trait works is similar to the musician trait, and makes it open for adding extra animal/anime bullshit as ear/tail organs.  It also means ANY race can take these traits and become cursed with anime sickness that has plagued this planet. Opens a lot of more customization options and whatnot for players.

TO DO:
- [x] make color choice easier(defaults to human hair choice)
- [x] replace the placeholder sprites
- [x] decide what to do with old organs getting dropped

Known issues/discussion
- Currently does not work properly with IPC. --Once KapuLimbs is finished this will be fixed.
- Currently doesn't show on Plasmamen through their helmet.  Could be fixed with adjusting the layers. 
- Decide whether or not it should conflict with other traits (wigs for example)
- Potentially add more options to initial trait (if anyone provides me sprites I will add them)
- wagging currently DOES NOT work, I know how to fix, but I want to wait for KapuLimbs to finish before I mess with that stuff.

## Testing Photographs and Procedure

<details>

<summary>Screenshots&Videos</summary>

image
image
image
image

</details>

## Changelog
:cl:
add: weeb trait
del: felinid species
/:cl:

``

### CODE REVIEW

File: code/__DEFINES/mobs.dm [hunk 1]
```diff
@@ -97,7 +96,6 @@
 #define	FLAG_APID			(1<<4)
 #define FLAG_MOTH			(1<<5)
 #define FLAG_LIZARD			(1<<6)
-#define FLAG_FELINID		(1<<7)
 #define FLAG_OOZELING		(1<<8)
 #define FLAG_FLY			(1<<9)
 #define FLAG_DEBUG_SPECIES	(1<<10)
```
Comment (nednaZ) [inline code/__DEFINES/mobs.dm:101]: Probably need to shift the rest and not leave a gap in the middle

File: code/datums/diseases/transformation.dm [hunk 2]
```diff
@@ -334,108 +334,6 @@
 				var/obj/item/I = affected_mob.get_active_held_item()
 				affected_mob.dropItemToGround(I)
 
-/datum/disease/transformation/felinid
```
Comment (nednaZ) [inline code/datums/diseases/transformation.dm:337]: This is actually an old felinid disease not related to the species

Comment (nednaZ) [reply code/datums/diseases/transformation.dm:337]: Plus if this is being removed, there is a string file that needs to be removed as well.
"strings/owo_talk.json"

Comment (polygoblyn) [reply code/datums/diseases/transformation.dm:337]: this does turn a player into a felinid species, i can make it just make you grow cat ears/tail instead

File: code/__DEFINES/mobs.dm (hunk 1 above)
Comment (polygoblyn) [reply code/__DEFINES/mobs.dm:101]: done

File: code/datums/diseases/transformation.dm (hunk 2 above)
Comment (nednaZ) [reply code/datums/diseases/transformation.dm:337]: Trashing the whole thing and making it an advanced symptom later might be best, especially with the words that owospeak gives you.

Comment (polygoblyn) [reply code/datums/diseases/transformation.dm:337]: Yeah, went ahead and removed it.

File: code/modules/mob/living/carbon/human/emote.dm [hunk 3]
```diff
@@ -151,7 +151,7 @@
 	if(!.)
 		return
 	var/mob/living/carbon/human/H = user
-	if(!istype(H) || !H.dna || !H.dna.species || !H.dna.species.can_wag_tail(H))
+	if(!istype(H) || !H.dna || !H.dna.species || !H.dna.species.can_wag_tail(H)) //Monkestation edit
```
Comment (nednaZ) [inline code/modules/mob/living/carbon/human/emote.dm:154]: What is the difference here that required the edit comment?

File: config/game_options.txt [hunk 4]
```diff
@@ -517,7 +516,7 @@ ROUNDSTART_RACES plasmaman
 ##-------------------------------------------------------------------------------------------
 ## Uncommenting races will restrict them behind the patreon paywall
 
-#PAYWALL_RACES felinid
+#PAYWALL_RACES none
```
Comment (nednaZ) [inline config/game_options.txt:519]: I think this isn't needed, just a total deletion of this line should work

File: monkestation/code/game/objects/items/anime.dm [hunk 5]
```diff
@@ -0,0 +1,70 @@
...
+	user.hair_color = sanitize_hexcolor(src.color) //I guess I have to do this fuck living code
+	if(ears)
+		ears.Insert(user)
+	if(tail)
+		tail.Insert(user)
+	user.hair_color = old_hair_color
+	var/turf/location = get_turf(user)
+	user.add_splatter_floor(location)
+	var/msg = "<span class=danger>You feel the power of God and Anime flow through you! </span>"
+	to_chat(user, msg)
```
Comment (nednaZ) [inline monkestation/code/game/objects/items/anime.dm:26]: Some kind of sound, probably taken from the autosurgeon, would make sense here.

File: code/modules/mob/living/carbon/human/emote.dm (hunk 3 above)
Comment (lmenvs) [reply code/modules/mob/living/carbon/human/emote.dm:154]: There is none, change was made elsewhere, deleting

File: config/game_options.txt (hunk 4 above)
Comment (lmenvs) [reply config/game_options.txt:519]: True, will be added back with Simian species in future PR anyways

File: monkestation/code/game/objects/items/anime.dm (hunk 5 above)
Comment (lmenvs) [reply monkestation/code/game/objects/items/anime.dm:26]: added

File: code/modules/admin/admin_verbs.dm [hunk 6]
```diff
@@ -91,6 +91,7 @@ GLOBAL_LIST_INIT(admin_verbs_fun, list(
 	/client/proc/cmd_admin_dress,
 	/client/proc/cmd_admin_gib_self,
 	/client/proc/drop_bomb,
+	/client/proc/give_disease,
```
Comment (nednaZ) [inline code/modules/admin/admin_verbs.dm:94]: Is this new, or left over from something else?

Comment (polygoblyn) [reply code/modules/admin/admin_verbs.dm:94]: its been in the code but the proc itself wasn't being used.  I forgot to remove it again but we could just leave it?

Comment (nednaZ) [reply code/modules/admin/admin_verbs.dm:94]: As long as it works and isn't just a broken proc, that's fine.

**Review by nednaZ — APPROVED** [review]
Looks good. RIP felinids.
We'll have IPC catgirls whenever Kapu finishes their limbs, so it's as functional as it can be right now.

**Comment by pitaden** [comment]
cant wait to make a plasmaman catgirl

**Pull request merged**